test

```diff
@@ -27,7 +27,7 @@ fi
 COVER=${COVER:-"-cover"}
 
 # Hack: gofmt ./ will recursively check the .git directory. So use *.go for gofmt.
-IGNORE_PKGS="(cmd|vendor|etcdserverpb|rafttest|gopath.proto|v3lockpb|v3electionpb)"
+IGNORE_PKGS="(cmd/vendor|vendor|etcdserverpb|rafttest|gopath.proto|v3lockpb|v3electionpb)"
```

Comment:
can this be done by updating `PKGS=` so it includes `tools` by removing the egrep mask? this will reprocess the entire repo since cmd/etcd points to the repo root